addons/links/src/preview.test.ts

```diff
@@ -48,9 +50,9 @@ describe('preview', () => {
     });
 
     it('should select the kind (only) provided', () => {
-      const channel = { emit: jest.fn() };
-      addons.getChannel.mockReturnValue(channel);
-      __STORYBOOK_STORY_STORE__.fromId.mockImplementation((input) => null);
+      const channel = { emit: jest.fn() } as any;
+      mockAddons.getChannel.mockReturnValue(channel);
+      __STORYBOOK_STORY_STORE__.fromId.mockImplementation((): any => null);
```

Comment:
This is not directly related to this PR but all the tests feel weird to me, we are always expecting the same thing i.e. `channel.emit` to have been called with `{ kind: 'kind', story: 'name' }` (which are the mocked values a the beginning of the file) whatever the input 🙃

Reply:
Before I saw your comment I saw this as well and went: "I must have forgot to reset mock, it doesn't make sense".
Definitely something weird here